source/CMakeLists.txt

```diff
@@ -85,17 +85,16 @@ add_library(
   util/TimeTrace.cpp)
 
 add_subdirectory(ast)
-
-set(SLANG_SHARED_LIB_NAME slang)
 add_library(slang::slang ALIAS slang_slang)
 set_target_properties(
   slang_slang
   PROPERTIES CXX_VISIBILITY_PRESET hidden
              VISIBILITY_INLINES_HIDDEN YES
              VERSION ${PROJECT_VERSION}
              SOVERSION ${PROJECT_VERSION_MAJOR}
-             EXPORT_NAME ${SLANG_SHARED_LIB_NAME}
-             OUTPUT_NAME ${SLANG_SHARED_LIB_NAME})
+             EXPORT_NAME ${SLANG_LIB_NAME}
+             OUTPUT_NAME ${SLANG_LIB_NAME})
+endif()
```

Comment:
There is a stray endif() here that is breaking the build.

Reply:
* Mea culpa, removed now.